app/models/miq_report/search.rb

```diff
@@ -104,6 +104,8 @@ def paged_view_search(options = {})
 
     if options[:parent]
       targets = get_parent_targets(options)
+      # filter out archived VMs
+      targets = targets.select { |target| !Vm.find_by(:id => target).archived? unless Vm.find_by(:id => target).nil? }
```

Comment:
@matthewd @Fryguy something looks off here (making the same SQL query twice on one line).  Suggestions?